src/applications/check-in/utils/appointment/index.js

```diff
@@ -110,12 +110,21 @@ const removeTimeZone = payload => {
 const preCheckinExpired = appointments => {
   return !Object.values(appointments).some(appt => {
     const today = new Date();
-    const checkInExpiry = startOfDay(new Date(appt.startTime));
-    return today.getTime() < checkInExpiry.getTime();
+    const preCheckInExpiry = startOfDay(new Date(appt.startTime));
+    return today.getTime() < preCheckInExpiry.getTime();
+  });
+};
+
+const appointmentStartTimePast15 = appointments => {
+  return !Object.values(appointments).some(appt => {
+    const today = new Date();
+    const deadline = add(new Date(appt.startTime), { minutes: 15 });
```

Comment:
I think we should probably use `checkInWindowEnd` rather than manually calculating 15 minutes after the start time, since VA might choose to change the window in the future.

Reply:
Updated